obspy/clients/syngine/client.py

```diff
@@ -0,0 +1,602 @@
+            [catalog]:[eventid]. The centroid time and location and moment
+            tensor of the solution will be used as the source.
+        :type eventid: str
+        :param sourcelatitude: Specify the source latitude.
+        :type sourcelatitude: float
+        :param sourcelongitude: Specify the source longitude.
+        :type sourcelongitude: float
+        :param sourcedepthinmeters: Specify the source depth in meters.
+        :type sourcedepthinmeters: float
+        :param sourcemomenttensor: Specify a source in moment tensor
+            components as a list: ``Mrr``, ``Mtt``, ``Mpp``, ``Mrt``, ``Mrp``,
+            ``Mtp`` with values in Newton meters (*Nm*).
+        :type sourcemomenttensor: list of floats
+        :param sourcedoublecouple: Specify a source as a double couple. The
+            list of values are ``strike``, ``dip``, ``rake`` [, ``M0`` ],
+            where strike, dip and rake are in degrees and M0 is the scalar
+            seismic moment in Newton meters (Nm). If not specified, a value
+            of *1e19* will be used as the scalar moment.
+        :type sourcedoublecouple: list of floats
+        :param sourceforce: Specify a force source as a list of ``Fr``, ``Ft``,
+            ``Fp`` in units of Newtons (N).
+        :type sourceforce: list of floats
+        :param origintime: Specify the source origin time. This must be
+            specified as an absolute date and time.
+        :type origintime: :class:`~obspy.core.utcdatetime.UTCDateTime`
+        :param starttime: Specifies the desired start time for the synthetic
+            trace(s). This may be specified as either:
+
+            * an absolute date and time
+            * a phase-relative offset
+            * an offset from origin time in seconds
+
+            If the value is recognized as a date and time, it is interpreted
+            as an absolute time. If the value is in the form
+            ``phase[+-]offset`` it is interpreted as a phase-relative time,
+            for example ``P-10`` (meaning Pwave arrival time minus 10
+            seconds). If the value is a numerical value it is interpreted as an
+            offset, in seconds, from the ``origintime``.
+        :type starttime: :class:`~obspy.core.utcdatetime.UTCDateTime`
+        :param endtime: Specifies the desired end time for the synthetic
+            trace(s). This may be specified as either:
+
+            * an absolute date and time
+            * a phase-relative offset
+            * an offset from origin time in seconds
+
+            If the value is recognized as a date and time, it is interpreted
+            as an absolute time. If the value is in the form
+            ``phase[+-]offset`` it is interpreted as a phase-relative time,
+            for example ``P-10`` (meaning Pwave arrival time minus 10
+            seconds). If the value is a numerical value it is interpreted as an
+            offset, in seconds, from the ``starttime``.
```

Comment:
Yes. Relative start times are relative to the origin time and relative end times relative to the start time. A bit confusing at first but actually nicer to use.